src/wallet/CarbonWallet.ts

```diff
@@ -151,14 +150,12 @@ export class CarbonWallet {
   triggerMerge?: boolean;
   publicKey: Buffer;
   query?: CarbonQueryClient;
+  gasFee?: GasFee;
 
   /**
    * @deprecated Carbon v2.9.0
    * use gasCosts and gasPrices instead.
    */
```

Comment:
Since you're removing `txFees`, `txGasCosts` and `txGasPrices`, can remove this deprecation notice as well.